lib/improver/nbhood.py

```diff
@@ -39,69 +39,51 @@
     concatenate_cubes)
 
 
-class BasicNeighbourhoodProcessing(object):
-    """
-    Apply a neigbourhood processing kernel to a thresholded cube.
-
-    When applied to a thresholded probabilistic cube, it acts like a
-    low-pass filter which reduces noisiness in the probabilities.
-
-    The kernel will presently only work with projections in which the
-    x grid point spacing and y grid point spacing are constant over the
-    entire domain, such as the UK national grid projection.
-
-    A maximum kernel radius of 500 grid cells is imposed in order to
-    avoid computational ineffiency and possible memory errors.
+class Utilities(object):
 
     """
+    Utilities for neighbourhood processing.
+    """
 
-    # Max extent of kernel in grid cells.
-    MAX_KERNEL_CELL_RADIUS = 500
+    def __init__(self):
+        pass
 
-    def __init__(self, radii_in_km, lead_times=None, unweighted_mode=False):
+    @staticmethod
+    def cumulate_array(cube):
```

Comment:
This method doesn't feel like a Utility. I would put this method into a class called `SquareNeighbourhood` with the method that uses the result of it (which I know isn't included in this ticket!).